pandas/tests/extension/base/getitem.py

```diff
@@ -40,6 +40,10 @@ def test_iloc_frame(self, data):
         result = df.iloc[:4, 0]
         self.assert_series_equal(result, expected)
 
+        # GH#32957 null slice along index, slice along rows
```

Comment:
```suggestion
        # GH#32957 null slice along index, slice along columns
```

? 
(or do I understand the "along" wrongly?)

Reply:
good catch, thanks